src/core/Table/Table.test.tsx

```diff
@@ -2445,3 +2445,189 @@ it('should not have `draggable` attribute on columns with `disableReordering` en
   expect(headerCells[3].getAttribute('draggable')).toBe('true'); // Description column
   expect(headerCells[4].getAttribute('draggable')).toBeFalsy(); // View column
 });
+
+it('should render empty action column with column manager', () => {
+  const columns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+        },
+        {
+          id: 'description',
+          Header: 'Description',
+          accessor: 'description',
+        },
+        {
+          id: 'view',
+          Header: 'View',
+          Cell: () => 'View',
+        },
+        ActionColumn({ hasColumnManager: true }),
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns,
+    data: mockedData(),
```

Comment:
Only columns is needed here, remove the remaining props.

Reply:
removed